src/alpha-lab/multiple-rim.md

```diff
@@ -4,93 +4,166 @@ permalink: /alpha-lab/multiple-rim/
-| <img src="../media/alpha-lab/kitchen+table-img.jpeg"/> |                                              |
-
-## Scanning recordings
-
-In this guide, we want to map gaze onto different parts of a living room, for this reason, we recorded **two** scanning videos. We chose to use more than one scanning recording because the environment is a bit too big to be effectively scanned just by a single one.
+## Exploring gaze patterns in multiple regions of an environment
+Understanding where people focus their gaze while exploring their environment is an area of interest for researchers in 
+diverse fields, ranging from art and architecture to medical training. Recently, powerful scene recognition tools such as 
+the Reference Image Mapper enrichment in Pupil Cloud have made it possible to map gaze in 3D real-world environments and 
+generate heatmap visualizations. These offer an informative overview of visual exploration patterns and also pave the way 
+for further analysis, such as region of interest analysis.
 
-Based on the environment dimension/complexity, you might need to do the same and record separate scanning videos.
-
-Please follow our [best practices](/enrichments/reference-image-mapper/#scanning-best-practices) for optimal scanning.
+In this guide, we will show you how to use the [Reference Image Mapper](/enrichments/reference-image-mapper/) to map a 
+participant's gaze onto _multiple_ areas of a living environment as they freely navigate around it.
 
 ::: tip
-To ensure good scanning of big plain surfaces - like tables and kitchen countertops - enrich them with features. Use a printed tablecloth and/or place items to produce a successful mapping!
+Before continuing, ensure you are familiar with the [Reference Image Mapper](/enrichments/reference-image-mapper) enrichment. 
+Check out [this explainer video](https://www.youtube.com/watch?v=ygqzQEzUIS4&t=56s) for reference.
 :::
 
-<div class="iframe-container2">
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/FQ2SdFcnqXw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+## The tools at hand
+The [Reference Image Mapper](/enrichments/reference-image-mapper/) enrichment available in Pupil Cloud can be used to map 
+gaze onto a _single_ reference image of an environment. However, mapping gaze onto *multiple* regions of an environment is 
+often of interest, since it can enable a deeper understanding of patterns of visual exploration.  Therefore, we have put 
+together this guide to show you how to leverage Reference Image Mapper for this purpose.
+
+## Steps
+Because the [Reference Image Mapper](/enrichments/reference-image-mapper/) is only able to map gaze onto a *single* reference image, 
+we need a few things to generate *multiple* reference image mappings in a larger environment:
```

Comment:
Strange formulation IMO. This guide does not really surpass the limitations of RIM, but rather it shows how apply RIM multiple times within the same physical space.